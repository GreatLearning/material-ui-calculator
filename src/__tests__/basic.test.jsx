import renderer from 'react-test-renderer';
import React from 'react';
import Basic from '../basic';

it('renders correctly', () => {
  const tree = renderer.create(<Basic onInput={jest.fn()} />).toJSON();
  expect(tree).toMatchSnapshot();
});
