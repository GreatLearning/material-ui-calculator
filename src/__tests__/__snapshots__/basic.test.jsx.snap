// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="Basic-pad-1"
>
  <div
    className="Pad-pad-3"
    style={Object {}}
  >
    <div
      dataName="IconButton"
    >
      <div
        dangerouslySetInnerHTML={
          Object {
            "__html": "C",
          }
        }
      />
    </div>
  </div>
  <div
    className="Pad-pad-3"
    style={Object {}}
  >
    <div
      dataName="IconButton"
    >
      <div
        dangerouslySetInnerHTML={
          Object {
            "__html": "Â±",
          }
        }
      />
    </div>
  </div>
  <div
    className="Pad-pad-3"
    style={Object {}}
  >
    <div
      dataName="IconButton"
    >
      <div
        dangerouslySetInnerHTML={
          Object {
            "__html": "%",
          }
        }
      />
    </div>
  </div>
  <div
    className="Pad-pad-3 Basic-operator-2"
    style={Object {}}
  >
    <div
      dataName="IconButton"
    >
      <div
        dangerouslySetInnerHTML={
          Object {
            "__html": "&#247;",
          }
        }
      />
    </div>
  </div>
  <div
    className="Pad-pad-3"
    style={Object {}}
  >
    <div
      dataName="IconButton"
    >
      <div
        dangerouslySetInnerHTML={
          Object {
            "__html": "7",
          }
        }
      />
    </div>
  </div>
  <div
    className="Pad-pad-3"
    style={Object {}}
  >
    <div
      dataName="IconButton"
    >
      <div
        dangerouslySetInnerHTML={
          Object {
            "__html": "8",
          }
        }
      />
    </div>
  </div>
  <div
    className="Pad-pad-3"
    style={Object {}}
  >
    <div
      dataName="IconButton"
    >
      <div
        dangerouslySetInnerHTML={
          Object {
            "__html": "9",
          }
        }
      />
    </div>
  </div>
  <div
    className="Pad-pad-3 Basic-operator-2"
    style={Object {}}
  >
    <div
      dataName="IconButton"
    >
      <div
        dangerouslySetInnerHTML={
          Object {
            "__html": "&#215;",
          }
        }
      />
    </div>
  </div>
  <div
    className="Pad-pad-3"
    style={Object {}}
  >
    <div
      dataName="IconButton"
    >
      <div
        dangerouslySetInnerHTML={
          Object {
            "__html": "4",
          }
        }
      />
    </div>
  </div>
  <div
    className="Pad-pad-3"
    style={Object {}}
  >
    <div
      dataName="IconButton"
    >
      <div
        dangerouslySetInnerHTML={
          Object {
            "__html": "5",
          }
        }
      />
    </div>
  </div>
  <div
    className="Pad-pad-3"
    style={Object {}}
  >
    <div
      dataName="IconButton"
    >
      <div
        dangerouslySetInnerHTML={
          Object {
            "__html": "6",
          }
        }
      />
    </div>
  </div>
  <div
    className="Pad-pad-3 Basic-operator-2"
    style={Object {}}
  >
    <div
      dataName="IconButton"
    >
      <div
        dangerouslySetInnerHTML={
          Object {
            "__html": "-",
          }
        }
      />
    </div>
  </div>
  <div
    className="Pad-pad-3"
    style={Object {}}
  >
    <div
      dataName="IconButton"
    >
      <div
        dangerouslySetInnerHTML={
          Object {
            "__html": "1",
          }
        }
      />
    </div>
  </div>
  <div
    className="Pad-pad-3"
    style={Object {}}
  >
    <div
      dataName="IconButton"
    >
      <div
        dangerouslySetInnerHTML={
          Object {
            "__html": "2",
          }
        }
      />
    </div>
  </div>
  <div
    className="Pad-pad-3"
    style={Object {}}
  >
    <div
      dataName="IconButton"
    >
      <div
        dangerouslySetInnerHTML={
          Object {
            "__html": "3",
          }
        }
      />
    </div>
  </div>
  <div
    className="Pad-pad-3 Basic-operator-2"
    style={Object {}}
  >
    <div
      dataName="IconButton"
    >
      <div
        dangerouslySetInnerHTML={
          Object {
            "__html": "+",
          }
        }
      />
    </div>
  </div>
  <div
    className="Pad-pad-3"
    style={
      Object {
        "gridColumn": "1/3",
      }
    }
  >
    <div
      dataName="IconButton"
    >
      <div
        dangerouslySetInnerHTML={
          Object {
            "__html": "0",
          }
        }
      />
    </div>
  </div>
  <div
    className="Pad-pad-3"
    style={Object {}}
  >
    <div
      dataName="IconButton"
    >
      <div
        dangerouslySetInnerHTML={
          Object {
            "__html": ".",
          }
        }
      />
    </div>
  </div>
  <div
    className="Pad-pad-3 Basic-operator-2"
    style={Object {}}
  >
    <div
      dataName="IconButton"
    >
      <div
        dangerouslySetInnerHTML={
          Object {
            "__html": "=",
          }
        }
      />
    </div>
  </div>
</div>
`;
